src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/entities/ReportedAdjudication.kt

```diff
@@ -113,6 +113,14 @@ enum class ReportedAdjudicationStatus {
   }
 
   fun canBeIssuedValidation() {
-    if (listOf(SCHEDULED, UNSCHEDULED).none { it == this }) throw ValidationException("$this not valid status for DIS issue")
+    if (issuableStatuses().none { it == this }) throw ValidationException("$this not valid status for DIS issue")
   }
+
+  companion object {
+    fun issuableStatuses() = listOf(SCHEDULED, UNSCHEDULED)
+  }
+}
+
+enum class IssuedStatus {
```

Comment:
Not a huge one.. I wasn't sure about this enum being here - it's _related_ to the `ReportedAdjudications` entity, but only used in `ReportsService`, maybe we could move it there, what do you think?